stack.yaml

```diff
@@ -1,31 +1,17 @@
-resolver: ghc-8.4
+resolver: lts-12.18
```

Comment:
Why not `lts-13.13` - that reflects nixpkgs master/19.03. Also gets rid of extra pinned versions (although I still think that servant-streaming will need to be removed).

Reply:
This is an incremental change. We'll do it in the near future.